tests/font_tests.ts

```diff
@@ -75,7 +74,7 @@ function fontParsing(assert: Assert): void {
   assert.equal(bFont?.weight, FontWeight.NORMAL);
   assert.equal(bFont?.style, FontStyle.NORMAL);
 
-  const f1 = 'Roboto Slab, serif';
+  const f1 = 'RobotoSlab, serif';
```

Comment:
Should it be 'Roboto Slab', with a space?

Reply:
done